### PR TITLE
docs: restore non-root podman section

### PR DESCRIPTION
**Pull request by mstopa-splunk**

### CODE REVIEW

File: docs/gettingstarted/podman-systemd-general.md [hunk 1]
```diff
@@ -118,3 +118,97 @@ starting syslog-ng
 
 If you do not see the output above, proceed to the ["Troubleshoot sc4s server"](../troubleshooting/troubleshoot_SC4S_server.md)
 and ["Troubleshoot resources"](../troubleshooting/troubleshoot_resources.md) sections for more detailed information.
+
+# SC4S non-root operation
+### NOTE: 
+Using non-root prevents the use of standard ports 514 and 601 many device can not alter their destination port this is not
```
[the remark this answers is not in the document]
Comment (mstopa-splunk) [reply docs/gettingstarted/podman-systemd-general.md:124]: @jenworthington can we rewrite the first sentence closer to this meaning? `Operating as a non-root user makes it impossible to use standard ports 514 and 601.`

File: docs/gettingstarted/podman-systemd-general.md [hunk 2]
```diff
@@ -118,3 +118,97 @@ starting syslog-ng
...
+a valid configuration for general use, and may only be appropriate for cases where accepting syslog from the public internet can not
+be avoided.
+
+## Prequisites
+Podman and slirp4netns installed.
+
+## Increase number of user namespaces
+With user that has sudo privileges:
```
[the remark this answers is not in the document]
Comment (mstopa-splunk) [reply docs/gettingstarted/podman-systemd-general.md:132]: Can we make it like `Execute with sudo privileges:`?

File: docs/gettingstarted/podman-systemd-general.md [hunk 3]
```diff
@@ -118,3 +118,97 @@ starting syslog-ng
...
+```bash
+sudo useradd -m -d /home/sc4s -s /bin/bash sc4s
+sudo passwd sc4s  # type password here
+sudo su - sc4s
+mkdir -p /home/sc4s/local
+mkdir -p /home/sc4s/archive
+mkdir -p /home/sc4s/tls
+podman system migrate
+```
+Next login as different user and login back again as sc4s user not using ```su``` command. For example: ```ssh sc4s@localhost``` (using `su` will not set needed env variables).
```
[the remark this answers is not in the document]
Comment (mstopa-splunk) [reply docs/gettingstarted/podman-systemd-general.md:150]: Can we make is something more like:
```
Next, you need to load the new environment variables. To do this, you can temporarily switch to any other user, and then log back in as sc4s. When logging in as sc4s, avoid using the su command, as it won't load the new variables. Instead, you can use, for example, the command ssh sc4s@localhost.
```

File: docs/gettingstarted/podman-systemd-general.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jenworthington) [reply docs/gettingstarted/podman-systemd-general.md:124]: Yeah, that works for me!

File: docs/gettingstarted/podman-systemd-general.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jenworthington) [reply docs/gettingstarted/podman-systemd-general.md]: Yes, that will work fine. :)

File: docs/gettingstarted/podman-systemd-general.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mstopa-splunk) [reply docs/gettingstarted/podman-systemd-general.md]: @jenworthington please take a look here too

**Pull request merged**